src/main/java/net/juniper/netconf/NetconfSession.java

```diff
@@ -37,6 +29,14 @@
 import java.util.List;
 import java.util.Map;
 import java.util.concurrent.TimeUnit;
+import javax.xml.parsers.DocumentBuilder;
```

Comment:
Not sure why you rearranged these? Any chance you could revert?

You'll need to keep the addition of the net....Hello; import. The others look like  you just moved them around.

Reply:
The re-arrange was my IDE being "helpful" - it's defaulting to alphabetical order. Apologies, I'll reformat to try and keep to the project style.

Reply:
I've also added a small `.editorconfig` file (https://editorconfig.org/) that in addition to some basic configs in line with the project, also define some IntelliJ specific configurations so that IntelliJ users, at least, won't trip up over the same thing!